src/components/AnnotationContentOverlay/AnnotationContentOverlay.js

```diff
@@ -22,10 +22,18 @@ const AnnotationContentOverlay = () => {
 
   useEffect(() => {
     const onMouseHover = e => {
-      const annotation = core
+      let annotation = core
         .getAnnotationManager()
         .getAnnotationByMouseEvent(e);
 
+      if (annotation) {
+        // if hovered annot is grouped, pick the "primary" annot
+        // do this as this is what Adobe does
+        const groupedAnnots = core?.getAnnotationManager()?.getGroupAnnotations(annotation);
```

Comment:
it came from a habit. i almost always put them.
since core is defined, and the functions being chained are most likely defined as well. i have removed them.